Library/Homebrew/test/dev-cmd/extract_spec.rb

```diff
@@ -29,4 +29,69 @@
 
     expect(Formulary.factory(path/"Formula/testball@0.1.rb").version).to be == "0.1"
   end
+
+  it "retrieves the specified version of formula from a tap other than core, defaulting to most recent" do
```

Comment:
Please don't add more integration tests for this, they are super slow. Either make this a unit test, alter the existing test or leave it untested, thanks.